packages/manager/modules/nasha/src/components/task-tracker/task-tracker.utils.js

```diff
@@ -0,0 +1,41 @@
+export const createTaskTrackerStateOptions = (queryParams = []) => ({
+  url: `/track-tasks?${['taskIds'].concat(queryParams).join('&')}`,
+  component: 'nashaComponentsTaskTracker',
+  params: {
+    tasks: null,
+  },
+  redirectTo: (transition) => {
+    const params = transition.params();
+    if (!params.taskIds) {
+      return {
+        state: transition.targetState().name(),
+        params: {
+          ...params,
+          taskIds: params.tasks.map(({ taskId }) => taskId).join(','),
+        },
+      };
+    }
+    return false;
+  },
+  resolve: {
+    endpoint: /* @ngInject */ (serviceName) =>
+      `/dedicated/nasha/${serviceName}/task`,
+    operation: /* @ngInject */ (tasks) => tasks[0].operation,
```

Comment:
why always the first one ? what if there are two tasks ?

Reply:
Because all the tasks have the same operation's value